app/helpers/requests/application_helper.rb

```diff
@@ -248,7 +248,7 @@ def submit_message(requestable_list)
     def submit_message_for_requestable_items(requestable_list)
       single_item = "Request this Item"
       multi_item = "Request Selected Items"
-      if requestable_list.first.annex? || requestable_list.first.pageable_loc?
+      if requestable_list.first.annex?
```

Comment:
Does this mean on items in the annex will ever see the "multi_item" submit message on line 250?

Reply:
Ah, good catch!  Should it be more like:
```suggestion
      if requestable_list.first.annex? || !requestable_list.first.charged?
```